docs/UserGuide.md

```diff
@@ -524,19 +559,35 @@ Action | Format, Examples
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Nuudle home folder.
 
 **Q**: What are the acceptable date time format?<br>
-**A**: Nuudle supports multiple date time formats as well as natural date time language :
-
-Date Formats | Time Formats | Natural Date | Natural Time
-:---------------:|:----------------:|:----------------:|:----------------:
-02/12/2020 | 2300 | Today | Morning (8AM)
-02-12-2020 | 11:00PM | Tomorrow | Noon (12PM)
-12/02/2020 | 11.00PM | Yesterday | Evening (7PM)
-12-02-2020 | 11PM | Upcoming day<br>of the week | Night (10PM)
-2020/12/02 | | | Midnight (11:59PM)
-2020-12-02 |
-02-Dec-2020 |
-02-December-2020 |
+**A**: Nuudle supports multiple date time formats as well as natural date time language. The details can be found [here](#command-format)
 
 **Q**: My data was accidentally deleted and I can't undo it. How do I restore my data?<br>
 **A**: Look for the `data` folder in the home directory of Nuudle and then look for a `backup` folder. Copy the 2 data files in the `backup` folder and paste them into the previous folder (`data`).
 The previous session's data will then be restored by overwriting the current data files with the backup data files.
+
+
+--------------------------------------------------------------------------------------------------------------------
+
+
+## Glossary
+
+This glossary provides definitions for the special terms used in the user guide.
+
+**Index:** The number on the left side of the corresponding entry in the patient or appointment book.
+
+**Tag:** A word or phrase that labels a patient.
```

Comment:
I think it would be more clear to say that a tag indicates the medical condition of a patient. What do you think?

Reply:
Good point!